apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -474,4 +474,13 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 			margin-bottom: 0;
 		}
 	}
+
+	// Adjust template title.
+	.editor-post-title__input {
+		overflow: hidden;
+		overflow-wrap: break-word;
+		resize: none;
+		max-height: 80px;
```

Comment:
This 80px seems risky. Why does it need to be 80px, and are we positive the title will never be multi-line? If the title goes multi-line, it would be hidden due to the max-height value.